docker/deployed/testnet/api/config.json

```diff
@@ -134,6 +134,34 @@
             },
             "HashCalculationStep": 150
         },
+        {
+            "Name": "Arbitrum Mainnet",
+            "ChainID": 42161,
+            "AllowTransactionRelay": false,
+            "Registry": {
+                "EthEndpoint": "https://arb-mainnet.g.alchemy.com/v2/${VALIDATOR_ALCHEMY_ARBITRUM_MAINNET_API_KEY}",
+                "ContractAddress": "<TBD>"
+            },
+            "Signer": {
+                "PrivateKey": "${VALIDATOR_ARBITRUM_MAINNET_SIGNER_PRIVATE_KEY}"
+            },
+            "EventFeed": {
+                "ChainAPIBackoff": "15s",
+                "NewBlockPollFreq": "5s",
+                "MinBlockDepth": 0,
+                "PersistEvents": true
+            },
+            "EventProcessor": {
+                "BlockFailedExecutionBackoff": "10s",
+                "DedupExecutedTxns": true
+            },
+            "NonceTracker": {
+                "CheckInterval": "20s",
+                "StuckInterval": "10m",
+                "MinBlockDepth": 0
+            },
+            "HashCalculationStep": 450
```

Comment:
Blocks are generated every 4s. Following the same line of thought as other configs: 450 = 30*60/4